client/src/components/exercise/lab9/Main.js

```diff
@@ -16,6 +17,7 @@ const Main = (props) => {
     <div className="bottomSpace">
       <GameStateContext.Provider>
         <Router className="app">
+          <Webpage path={"page"} />
```

Comment:
Will need to update once we have more of the exercise routing done

Reply:
👍🏼